src/vs/workbench/browser/parts/views/media/views.css

```diff
@@ -17,6 +17,14 @@
 
 /* Misc */
 
+.file-icon-themable-tree .monaco-list-row .content .monaco-highlighted-label .highlight,
+.monaco-tl-contents .monaco-highlighted-label .highlight {
+	color: unset !important;
+	background-color: var(--vscode-list-filterMatchBackground);
+	border: 1px dotted var(--vscode-list-filterMatchBorder);
+	box-sizing: border-box;
+}
+
```

Comment:
This isn't equivalent to the previous state, did you mean to not consider textLinkForeground, focusBorderColor, codeBackground?

Reply:
Thanks for noticing! It is intentional. The other colors are all related to having markdown in the tree view message. At one time, before the message API was finalized, we did allow markdown in the message. However, the finalized API does not allow markdown, and there are no plans to allow markdown here.